tests/Fixer/FunctionNotation/NoSpacesAfterFunctionNameFixerTest.php

```diff
@@ -106,6 +106,18 @@ public function provideFixCases()
             [
                 '<?php include ($html)? "custom.html": "custom.php";',
             ],
+            [
+                '<?php $r = require ($dir ?: __DIR__) . "/file.php";',
+            ],
+            [
+                '<?php $one = print (2 + 3) / 4;',
+            ],
+            [
+                '<?php echo ($a ?: $b) instanceof Foo;',
+            ],
+            [
+                '<?php echo ($a === $b), "\n";',
```

Comment:
Note: in this case the parentheses are actually unnecessary, but so are they in several of the existing cases above